commands/saml_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"github.com/mattermost/mattermost-server/v5/model"
+
+	"github.com/mattermost/mmctl/printer"
+
+	"github.com/spf13/cobra"
+)
+
+func (s *MmctlUnitTestSuite) TestSamlAuthDataReset() {
+	s.Run("Reset auth data without errors", func() {
+		printer.Clean()
+		outputMessage := "1 user records were changed.\n"
+
+		s.client.
+			EXPECT().
+			ResetSamlAuthDataToEmail(false, false, []string{}).
+			Return(int64(1), &model.Response{Error: nil})
```

Comment:
nit: adding `Times` call to keep consistency with the rest of the tests

```suggestion
			Return(int64(1), &model.Response{Error: nil}).
			Times(1)
```